app/src/main/java/com/oxeanbits/forecastchart/model/LineDataCreator.kt

```diff
@@ -0,0 +1,10 @@
+package com.oxeanbits.forecastchart.model
+
+import com.github.mikephil.charting.data.Entry
+
+data class LineDataCreator (
```

Comment:
```suggestion
data class LineData (
```

**or** 

```suggestion
data class Line (
```

Reply:
`LineData` is already a chart library model.